src/pyscipopt/conshdlr.pxi

```diff
@@ -259,8 +259,7 @@ cdef SCIP_RETCODE PyConsSepasol (SCIP* scip, SCIP_CONSHDLR* conshdlr, SCIP_CONS*
     cdef constraints = []
     for i in range(nconss):
         constraints.append(getPyCons(conss[i]))
-    solution = Solution()
-    solution.sol = sol
+    solution = Solution.create(scip, sol)
```

Comment:
My only concern with the switch to `Solution.create` is that it raises and error if `scip` is NULL, which it is not the same as the current behavior. @fserra, @mattmilten is that ok? See `src/pyscipopt/sepa.pxi` as well.

Reply:
I would argue that SCIP shouldn't be NULL... if it is NULL I guess there are bigger problems